provisioning/Build.ps1

```diff
@@ -0,0 +1,63 @@
+<#
+  This script installs the dependencies for gpii-app:
+  1) installs nodejs via chocolatey
+  2) installs the gpii-app npm dependencies
+  3) runs the provisioning scripts from the (gpii-)windows repo
+
+  If run via a tool (like vagrant) which moves this script to somewhere different
+  than its original location within the gpii-app repository, the parameter
+  "-originalBuildScriptPath" should be provided, with the original location of the
+  script
+#>
+
+param (
+    [string]$originalBuildScriptPath = (Split-Path -parent $PSCommandPath) # Default to script path.
+)
+
+# Turn verbose on, change to "SilentlyContinue" for default behaviour.
+$VerbosePreference = "continue"
+
+# Store the parent folder of the script (root of the repo) as $mainDir
+############
+$mainDir = (get-item $originalBuildScriptPath).parent.FullName
+Write-OutPut "mainDir set to: $($mainDir)"
+
+# TODO: We should add this to a function or reduce to oneline.
+$bootstrapModule = Join-Path $originalBuildScriptPath "Provisioning.psm1"
+iwr https://raw.githubusercontent.com/GPII/windows/hst-2017/provisioning/Provisioning.psm1 -UseBasicParsing -OutFile $bootstrapModule
+Import-Module $bootstrapModule -Verbose -Force
+
+# # Run all the windows provisioning scripts
+# ############
+# TODO: Create function for downloading scripts and executing them.
+$windowsBootstrapURL = "https://raw.githubusercontent.com/GPII/windows/hst-2017/provisioning"
+try {
+    $choco = Join-Path $originalBuildScriptPath "Chocolatey.ps1"
+    Write-OutPut "Running windows script: $choco"
+    iwr "$windowsBootstrapURL/Chocolatey.ps1" -UseBasicParsing -OutFile $choco
```

Comment:
I assume that an overwrite of the file will occur when running provisioning again but we should delete it after the provisioning process ends. Can you please note it down somewhere so this gets done in your next iteration? Thanks